backend/src/api/article/middlewares/isAuth.js

```
@@ -0,0 +1,136 @@
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+        </p>
+      </div>`;
+
+    // Append the new <div> tag after the clipped content
+    clippedContent += blurredParagraph;
+
+    ctx.response.body.Content = clippedContent;
+  }
+
+  // Remove recommended articles section if present
+  delete ctx.response.body.recommended;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)